docs/docs/administration/install-from-scratch.md

```diff
@@ -223,19 +225,20 @@ Then press <kbd>Enter</kbd> to customize this guide with your domain name.
     ```
 
 
-1. Install Mathesar functions on the database:
+1. Create a media directory for storing user-uploaded media
 
     ```
-    python -m mathesar.install --skip-confirm | tee /tmp/install.py.log
+    mkdir .media
     ```
 
 
-1. Create a media directory for storing user-uploaded media
+1. Install Mathesar functions on the database:
```

Comment:
This description is no longer accurate. This step now runs the Django migrations and collects static files.

Reply:
Fixed in https://github.com/mathesar-foundation/mathesar/pull/4191/commits/2b90b066713c4f4edf7eb02c5c0fac9e0df328fc